features/storage/csi_clone.feature

```diff
@@ -0,0 +1,58 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use "oc get sc" can find all deployed stroage class.
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
+  Scenario: Clone a pvc on OSP in the same namespace is successfully
```

Comment:
Grammar error, rename scenario: `Clone a PVC and verify data consistency`

Reply:
@jhou1 um...I used the description of the test case...but you are right, we should use correct English grammar.